documentation-site/examples/progress-steps/dotted.js

```diff
@@ -2,19 +2,25 @@
 import * as React from 'react';
 import {useStyletron} from 'baseui';
 import {ProgressSteps, Step} from 'baseui/progress-steps';
-import {Button} from 'baseui/button';
+import {Button, SHAPE} from 'baseui/button';
 import {StatefulSelect, TYPE} from 'baseui/select';
 
 function SpacedButton(props) {
   return (
     <Button
       {...props}
+      shape={SHAPE.pill}
       overrides={{
         BaseButton: {
           style: ({$theme}) => ({
             marginLeft: $theme.sizing.scale200,
             marginRight: $theme.sizing.scale200,
-            marginTop: $theme.sizing.scale200,
+            marginTop: $theme.sizing.scale800,
+            width: $theme.sizing.scale1800,
+            height: $theme.sizing.scale900,
```

Comment:
width/height/color/backgroundColor/font overrides are a bit concerning. Can those styles be configured through button `kind` and `size` instead?

Reply:
Sure!